random/between.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
+import type { RandomOptions } from "./_types.ts";
+
+/**
+ * Generates a random number between the provided minimum and maximum values.
+ *
```

Comment:
Please ensure that all public symbols follow [this guideline](https://github.com/denoland/std/blob/main/.github/CONTRIBUTING.md#notices-for-unstable-apis).